build.sh

```diff
@@ -226,12 +226,12 @@ if buildAll || hasArg ucxx; then
 
     cd ${REPODIR}/python/
     SKBUILD_CMAKE_ARGS="-DCMAKE_PREFIX_PATH=${INSTALL_PREFIX};-DCMAKE_BUILD_TYPE=${BUILD_TYPE};${SKBUILD_EXTRA_CMAKE_ARGS}" \
-        python -m pip install --no-build-isolation --no-deps .
+        python -m pip install --no-build-isolation --no-deps --config-settings rapidsai.disable-cuda=true .
```

Comment:
Sorry if the name is confusing. This does NOT disable linking against anything from the CUDA Toolkit.

This option tells `rapids-build-backend` to avoid doing the following:

* add a suffix like `-cu11` or `-cu12` to the name of the package it builds
* update the list of dependencies by reading the output of running `rapids-dependency-file-generator` with something like `cuda="12.2"`

It's passed here in `build.sh` for conda builds because the conda packages we publish don't have `-cu{version}` type of suffixes.

Reply:
Got it, thanks for the details. In that case we're good here! 😄